client.go

```diff
@@ -107,7 +115,7 @@ func (c *Client) Call(ctx context.Context, service, method string, req, resp int
 		creq.TimeoutNano = dl.Sub(time.Now()).Nanoseconds()
 	}
 
-	if err := c.dispatch(ctx, creq, cresp); err != nil {
+	if err := c.interceptor(ctx, creq, cresp, c.dispatch); err != nil {
```

Comment:
Can you add a `UnaryClientInfo` passed in here that contains the `FullMethod` name? The tracing interceptors use this to set the span name on the client side.

Reply:
added